ios/AppDelegate.m

```diff
@@ -43,7 +44,15 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   [[ExposureManager shared] registerBackgroundTask];
 
   [RNSplashScreen showSplash:@"LaunchScreen" inRootView:rootView];
+  
+  // Read dev environment variable
+#if DEBUG
```

Comment:
@mattThousand or @emilianojuan , can i get a quick gut check on this bit of code?

Reply:
@johnschoeman yes this is what we want I believe - bugsnag will be enabled only for builds either running locally using the `debug` build configuration, or any build using `.env.staging`

Reply:
I think its ok maybe add @"STAGING" as a static string but you may have issues with swift/objc so its ok to me, also you can do [ReactNativeConfig envFor:@"STAGING"].boolValue if you dont want to use the Boolean literal

Reply:
awesome, thanks!